docs/build-on-polkadot/parachains/asset-hub/register-a-local-asset.md

```diff
@@ -0,0 +1,90 @@
+    - `asset name` - the descriptive name of the asset you are registering
+    - `asset symbol` - the symbol that will be used to represent the asset
+    - `asset decimals` - the number of decimal places for this token, with a maximum of 20 allowed through the user interface
+    - `minimum balance` - the minimum balance for the asset. This is specified in the units and decimals as requested
+    - `asset ID` - the selected id for the asset. This should not match an already-existing asset id
+ 
+    ![Create Asset Form](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/registering-a-local-asset/registering-a-local-asset-4.webp)
+
+6. Select the `admin account`, `issuer-account` and `freezer-account`, and then click on the **Create** button
+
+    - `admin account` - the account designated for continuous administration of the token      
+    - `issuer account` - the account that will be used for issuing this token
+    - `freezer account` - the account that will be used for performing token freezing operations
+
+    ![Admin, Issuer, Freezer accounts](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/registering-a-local-asset/registering-a-local-asset-5.webp)
+
+8. Finally, click on the **Sign and Submit** button to complete the asset registration process.
+
+    ![Sign and Submit](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/registering-a-local-asset/registering-a-local-asset-6.webp)
+
+## Verify asset registration
+
+After completing these steps, the asset will be successfully registered. You can now view your asset listed on the [`Assets`](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fasset-hub-polkadot-rpc.dwellir.com#/assets){target=_blank} section of the Polkadot.js Apps interface.
+      
+![Asset listed on Polkadot.js Apps](/polkadot-ecosystem-docs-draft/images/building-on-polkadot/parachains/asset-hub/registering-a-local-asset/registering-a-local-asset-7.webp)
+
+!!! note
+    Take into consideration that the `Assets section` link will be different according to the network you are using. For the local environment, the link will be `ws://127.0.0.1:8000/#/assets`.
+
+In this way, you have successfully registered a local asset on the Asset Hub parachain.
+
+For an in-depth explanation of Asset Hub and its features, please refer to the Polkadot Wiki page on [Asset Hub](https://wiki.polkadot.network/docs/learn-assets){target=\_blank}.
+
+## Environment setup
+
+It might be beneficial to set up a local parachain environment to test the asset registration process before deploying it on the live network. This guide uses `chopsticks` to simulate that process. For further information on chopsticks usage, refer to the [Chopsticks](../../../dev-tools/chopsticks/overview.md){target=_blank} documentation.
+
+To set up a test environment then, execute the following command:
```

Comment:
Is something missing from this sentence? If not, please take out 'then' and the comma.